velox/functions/CoverageUtil.h

```diff
@@ -17,9 +17,19 @@
 
 #include <string>
 #include <unordered_set>
+#include <vector>
 
 namespace facebook::velox::functions {
 
+// Returns sorted list of scalar function names available in Velox.
+std::vector<std::string> getSortedScalarNames();
```

Comment:
@pramodsatya : We shouldn't use functions from CoverageUtil.h. You should move them to a common place. FunctionRegistry.h is probably best.

Reply:
Thanks for the feedback, moved them to `FunctionRegistry.h`.